metrix/src/models.rs

```diff
@@ -1,7 +1,21 @@
-#[derive(Queryable)]
+use super::schema::metrics;
+use serde_json;
+use std::time::SystemTime;
+
+#[derive(Serialize, Deserialize, Queryable)]
+// #[table_name="metrics"] -- idk why it throws compile error
+// "error: cannot find attribute `table_name` in this scope" ...
 pub struct Metric {
     pub id: i32,
     pub metric_name: String,
-    pub body: String,
-    pub created_at: String,
+    pub data: serde_json::Value,
+    pub created_at: SystemTime,
```

Comment:
Oh my, does this automatically serialize to
```
"created_at": {
        "secs_since_epoch": xxxxxxxx,
        "nanos_since_epoch": xxxxxxxxx
    }
```?
That's uggo

Reply:
the timestamp serialization isn't ideal...probably just need to change the attr type, see: https://docs.diesel.rs/diesel/sql_types/struct.Timestamp.html

Reply:
`NaiveDateTime` maybe?

Reply:
Yea, just did it. It's _another_ dependency...